apps/cli/src/web/index.ts

```diff
@@ -52,10 +61,10 @@ export async function install(session: ISession): Promise<void> {
   }
   await makeExecutable('npm install', createNpmLogger(session), { cwd: session.repoPath() })();
   session.log.info(toc('📦 Installed web libraries in %s'));
-  await makeExecutable('npm run build:web', createNpmLogger(session), {
-    cwd: session.repoPath(),
-  })();
-  session.log.info(toc('🛠  Built dependencies in %s'));
+  // await makeExecutable('npm run build:web', createNpmLogger(session), {
```

Comment:
Just kill this commented stuff now?

Reply:
I left it in because we will need to build in future, but that build step is dependent on the template.